src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/AbstractFunctional.java

```diff
@@ -58,7 +59,7 @@
     @Before
     public void setupProxy() {
         //TO enable for local testing
-        /*RestAssured.proxy("proxyout.reform.hmcts.net",8080);
+        /* RestAssured.proxy("proxyout.reform.hmcts.net",8080);
```

Comment:
Could revert the two changes in this file instead of committing them

Reply:
we need this code to run the functional test cases locally so thats why we have commented the code

Reply:
I know :) I just meant revert the two changes (the extra spaces) in the file rather than committing it (small change - not required, just a suggestion)